homeassistant/components/verisure/lock.py

```diff
@@ -75,6 +76,22 @@ def name(self) -> str:
         """Return the name of the lock."""
         return self.coordinator.data["locks"][self.serial_number]["area"]
 
+    @property
+    def unique_id(self) -> str:
+        """Return the unique ID for this entity."""
+        return f"{self.serial_number}"
```

Comment:
Serial number is already a string, no need to wrap in format-string.

```suggestion
        return self.serial_number
```